gammapy/data/obs_table.py

```diff
@@ -74,7 +74,12 @@ def select_obs_id(self, obs_id):
 
     def summary(self):
         """Summary info string (str)."""
-        obs_name = self.meta.get("OBSERVATORY_NAME", "N/A")
```

Comment:
I tend to agree with @AtreyeeS . If there is no keyword specification for the observatory name in the specifications, we shouldn't try to guess one. 
I would competely remove this part.

Reply:
Agree, even if the gammapy team and friends are leading the data formatting. It is not 'others' that will make specifications.